### PR TITLE
Add hyc v0.15.0.0

### PR DESCRIPTION
**Pull request by hyc**

### CODE REVIEW

**Comment by TheCharlatan** [comment]
Thanks!

File: v0.15.0.0-linux/hyc/monero-osx-0.15-build.assert [hunk 1]
```diff
@@ -0,0 +1,376 @@
+--- !!omap
+- out_manifest: 'bca6b776e7e906fcda4c829aa8666eb9d92014450b87a6723a7c6eda6d6e7de1  monero-x86_64-apple-darwin11-v0.15.0.0.tar.bz2
```
Comment (jonathancross) [inline v0.15.0.0-linux/hyc/monero-osx-0.15-build.assert]: This duplicate of the osx assert should be be removed.

**Comment by TheCharlatan** [comment]
Could you sign your commit?

File: v0.15.0.0-linux/hyc/monero-osx-0.15-build.assert (hunk 1 above)
Comment (hyc) [reply v0.15.0.0-linux/hyc/monero-osx-0.15-build.assert]: Fixed

**Comment by hyc** [comment]
Signed commit pushed

**Pull request merged**